### ISSUE TITLE
mouse hover on username causes line-break

### ISSUE THREAD

**Issue by mikrohash** [orderedlist/minimal#42]
The username in the footer paragraph "This project is maintained by {USERNAME}" appears bold on hover. However, depending on the length of the username, these few extra pixels might make the username become too big and cause a line-break. Here is an [example](https://mikrohash.github.io/isf-jclient/) which might not work in every browser (since this is pixel work), therefore I attached a demonstration.
![hover](https://user-images.githubusercontent.com/33261108/36634249-eb044e4a-19a1-11e8-8c31-a07035e12092.png)
![no hover](https://user-images.githubusercontent.com/33261108/36634250-eb24f244-19a1-11e8-825a-6440b8be3984.png)

**Comment by contactrika** [comment]
Agreed, this was a very problematic issue! I think I found a fix for this:
https://github.com/orderedlist/minimal/issues/44#issuecomment-826057962
